### ISSUE TITLE
Pipeline Running extra control:sample comparisons that are not needed

### ISSUE THREAD

**Issue by slsevilla** [CCBR/CARLISLE#48]
Email from Yue
```

  I think there might be a bug for output of seacr/goPeaks output of cut&run pipeline when there are multiple biological replicates, and each replicate having one control like in the following samples.csv :
Image
 
The experiment was done twice, one with NT_control_siRNA_Rb_CENPA as control, another with NT_control_siRNA_Ms_CENPA as control. There should be 4 outputs for 4 non_control samples.  I found seacr and goPeaks will have 8 output instead: each non_control sample will be compared with both controls, e.g., CHAF1B_siRNA_Rb_IgG will be compared with both NT_control_siRNA_Rb_CENPA AND NT_control_siRNA_Ms_CENPA.
 
Macs2 seems to be fine.
 
The result is in /data/khanlab3/gb_omics/YueHu/output/cutrun/008_Zhigang_011823/Dan_t2t_021723
 
Let me know if I have had any misunderstanding.
 
Thank you,
 
Yue
```

**Comment by slsevilla** [comment]
Will create additional output file to confirm the sample to control contrasts to be performed.

**Comment by slsevilla** [comment]
fixed with commit e19fe50f980e9c852d1c240876f46d0649ec3b15 d957f2e9adc55c594d139f38e9bdc0fc58573f4f
